docs/common_links.txt

```diff
@@ -65,7 +65,7 @@
 .. Units
 .. |PhysicalType| replace:: :class:`~astropy.units.PhysicalType`
 .. |Quantity| replace:: :class:`~astropy.units.Quantity`
-.. |Unit| replace:: :class:`~astropy.units.UnitBase`
+.. |Unit| replace:: :class:`~astropy.units.Unit`
```

Comment:
This is a tricky one. Certainly `Unit` is better if one wants to *initialize* a unit, but `UnitBase` has all the relevant attributes and methods, so is better if one wants to *use* a unit. Though I think that could be solved by a properly used `:inherited-members:` in the `automodapi` entry.

Anyway, overall this seems substantially more logical.

Reply:
@eerovaher , do you wish to accept this suggestion?

Reply:
It might be better to come back to this after the feature freeze rush is over, so I've removed the commit that made this change.

Reply:
Sounds good. I will merge then. Thanks!